src/js/components/modals/AlertModalComponent.jsx

```diff
@@ -12,7 +13,15 @@ var AlertModalComponent = React.createClass({
   },
 
   componentDidMount: function () {
-    React.findDOMNode(this.refs.button).focus();
+    let refs = this.refs;
+
+    React.findDOMNode(refs.button).focus();
+    Mousetrap(React.findDOMNode(refs.modalComponent))
+      .bind("esc", this.handleDestroy);
```

Comment:
Unfortunately we have no other choice than adding an eslint ignore directive here, as we can't just use `.bind("ESC")` as it attaches the handler to the `document`, which will override the global ESC for all the other dialogs (see commit message).